java/data-management/arrays/copying-an-array-with-system-arraycopy.md

```diff
@@ -2,54 +2,118 @@
 author: adamMontgomerie
 
 levels:
-
   - basic
-
   - advanced
-
   - medium
 
 type: normal
 
 category: tip
 
 aspects:
-
   - introduction
-
   - obscura
 
 tags:
-
   - loops
-
   - arrays
 
 links:
-
   - '[www.javapractices.com](http://www.javapractices.com/topic/TopicAction.do?Id=3){website}'
 
 ---
 
-# Copying an array with `System.arraycopy()`
+# Copying an Array
 
 ---
 ## Content
 
-`System.arraycopy` can be used to copy the contents of an array to a new location in a more concise way than using a loop. For example, to copy content from `src` to `dest.`
+To copy the contents of an array, you can use the  `Arrays.copyOf()` method. It can be found in the `java.util.Arrays` class, and the syntax is:
+
+```java
+import java.util.Arrays;
+
+copyOf(String[] original, int newLength);
+```
+
+Notice how we've used the `String` type for the our array. This is not require, the array can have any type. Make sure to change the type depending on your array.
```

Comment:
This paragraph:
Notice how we've used the `String` type for the our array. This is not require, the array can have any type. Make sure to change the type depending on your array.

Should be:
Notice how we've used the `String` type for our array. This is not required, the array can have any type. Make sure to change the type depending on your array.

Reply:
```suggestion
Notice how we've used the String type for our array. This is not required, the array can have any type. Make sure to change the type depending on your array.
```